lib/deep_cover/instrumented_clone_reporter.rb

```diff
@@ -141,6 +141,12 @@ def patch
       patch_main_ruby_files
     end
 
+    def remove_deep_cover_config
```

Comment:
TrackerBucket used to always provide a new index based on its current size when asked. We now always start at 0 and increment.
If the globals were clashing, it could seamlessly not fail (and hide the problem at the same time). Now it's stricter and so less random-prone errors.